airbyte-bootloader/Dockerfile

```diff
@@ -1,10 +1,14 @@
 ARG JDK_VERSION=17.0.1
 FROM openjdk:${JDK_VERSION}-slim
 
+ARG VERSION=0.35.60-alpha
```

Comment:
why do we need the arg at all? (not a leading question, i genuinely am unsure of the purpose it serves. would it make sense to always pull the version as an arg that must be passed when we build?)

Reply:
I think we could definitely go that route if we wanted to- the approach I took here with the defaulting behavior was to preserve existing behavior/dev workflows as much as possible. I wanted things to 'just work' if you run `SUB_BUILD=PLATFORM ./gradle build`

Reply:
legit. okay. i think my head is gravitating to a separate refactor. seems like we _should_ be able to have `SUB_BUILD=PLATFORM ./gradle build` "just work" and have gradle be in charge of passing the version in without having to hard code it in the dockerfile. 

that said, we should _not_ block on this!

Reply:
Totally agreed!